system/topic_state_monitor/include/topic_state_monitor/topic_state_monitor.hpp

```diff
@@ -31,6 +31,7 @@ struct Param
   double error_rate;
   double timeout;
   int window_size;
+  bool transient_local;
```

Comment:
@tkimura4 LGTM, but I have one request for you.
Could you put this under `topic_type`? It's for gathering topic-related parameters in one place.

Reply:
Fixed at 5820c61.